### ISSUE TITLE
Changes in Topic Fragments [A11y]

### ISSUE THREAD

**Issue by Arjupta** [oppia/oppia-android#2295]
**Describe the bug**

Add Profile Activity is not fully compatible with some users from the Accessibility point of view.

![Screenshot_2020-12-17-10-55-15-07_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102745782-c5e71f80-4382-11eb-85a3-33c62e94c6c4.jpg)

Issue 1 - The contrast ratio of the Tab Icon text (For inactive Tabs) of the Tab Bar in **topic_fragment.xml** is not correct as mentioned in the screenshot (assigned to @ARJUPTA)

Issue 2 - The inactive Topics, displayed in  **topic_lesson_story_summary.xml** does not have correct contrast ratio. (assigned to @ARJUPTA)

Issue 3 - The Button in the "**topic_practice_footer_view.xml** does not have the correct contrast ratio. It must be marked disabled instead of non-clickable only. (assigned to @ARJUPTA)

Issue 4 - The recycler view inside **topic_revison_fragment.xml** Does not have a content label (Obsolete Issue)

**To Reproduce**
Steps to reproduce the behaviour:
1. Start the Accessibility Scanner App on your device
2. Open the Oppia Android App and start First Test Topic Exploration
3. Scan the screenshots with the Accessibility Scanner
4. You will see the tests that are not passed by that screen
5. Repeat this for other screens

**Expected behaviour**
All the tests must be passed by scanning the screenshots for both Tablet and Mobile Devices in every orientation.

**Screenshots**

| Issue 1 | Issue 2 | Issue 3 | Issue 4 |
|:-------:|:-------:|:-------:|:-------:|
|![Screenshot_2020-12-17-10-55-57-55_80a9469f53bfe75dbc6e8989be28e2f0 (1)](https://user-images.githubusercontent.com/64064110/102746775-f4fe9080-4384-11eb-92b8-0b978b9993d3.jpg) | ![Screenshot_2020-12-17-11-04-04-98_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102747060-88d05c80-4385-11eb-8b0f-7fd25d16f7f6.jpg) |![Screenshot_2020-12-17-10-56-09-35_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102746910-38f19580-4385-11eb-8920-96678d76f656.jpg)| ![Screenshot_2020-12-17-10-55-37-57_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102746839-13648c00-4385-11eb-9cf0-6c7f235e7804.jpg) 
| - | ![Screenshot_2020-12-17-11-03-28-30_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102747084-91c12e00-4385-11eb-8dd9-f87f757c682c.jpg) | - | - | 

**Device**
- Real Device - RealMe3
- SDK version - 29

**Additional context**
The Screenshots are descriptive enough for understanding the issues but still, if there is a need of clarification then you can comment here. Multiple Pull Requests can also be made for solving these issues.

Certain Views does not have the correct contrast ratios (Inactive state) as mentioned in the screenshots. To solve this, one must find a way to make the views disabled instead of being in non-clickable state only

Reference - [https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner](https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner)

**Comment by Arjupta** [comment]
### Checklist
 (Contributors can take one issue at a time)
* [x]  `Issue-1` (assigned to @ARJUPTA)
* [x]  `Issue-2` (assigned to @ARJUPTA)
* [x]  `Issue-3` (assigned to @ARJUPTA)
* [ ]  `Issue-4` (not assigned to anyone)

**Comment by Arjupta** [comment]
Reopening it as it got closed automatically by merging a PR which solved only a part of this Issue

**Comment by Arjupta** [comment]
@rt4914 should we solve the Issue 4 by marking it not important for accessibility if the recycler view is empty?

**Comment by rt4914** [comment]
@ARJUPTA No, infact this issue is obsolete and won't happen. This is happening just because of dummy data else we will have subtopics in correct data.

**Comment by Arjupta** [comment]
@rt4914 Updated the description and heading

**Comment by rt4914** [comment]
Closing this issue as issue 4th is obsolete. Thanks @ARJUPTA . Also please verify that we can close this issue.

**Comment by Arjupta** [comment]
> Closing this issue as issue 4th is obsolete. Thanks @ARJUPTA . Also please verify that we can close this issue.

Tested it on Pixel 3A (API 29) and Nexus 9A (API 29) emulator for both landscape and portrait orientation. It can be closed
